packages/expo-modules-core/android/src/main/java/expo/modules/kotlin/AppContext.kt

```diff
@@ -151,6 +152,12 @@ class AppContext(
   val filePermission: FilePermissionModuleInterface?
     get() = legacyModule()
 
+  /**
+   * Provides access to the scoped directories from the legacy module registry.
+   */
+  val directories: DirectoriesModuleInterface?
```

Comment:
Since the directories module has only one thing, maybe it's better to get these directories right away from here instead of then referring to it as `directories.directories` 😅 
I would also think about changing the name to "app directories"; "directories" is too vague.

Reply:
Yeah, `appContext.cacheDirectory` and `appContext.documentDirectory` would be great 😉 (for consistency let's use the same names as FileSystem module)